_careers/big_data_eng.md

```diff
@@ -2,40 +2,45 @@
 layout: default
 title: Big Data Engineer
 location: "Prague, CZ"
-excerpt: "As a Big Data Engineer you will work alongside experienced and junior engineers on development of mission critical projects and applications using the most modern Big Data technology stack."
+excerpt: "As a Big Data Engineer you will work alongside experienced and junior engineers on development of mission critical projects and applications using the most modern Big Data technology stack. This role has a high potential for further career progress in the direction of technical leadership team-wide and/or big data architecture."
 ---
 Location: {{ page.location }}
 
 ### About the role
 {{ page.excerpt }}
 
 ### Responsibilities
-- Design and implementation of generic tools and platforms for some of the following:
-    - enabling simple data ingestion and consumption of corporate data lake
-    - various streaming use cases and products
-    - machine learning applications and pipelines
-- Contribute to our projects, which can be found on [GitHub](https://github.com/AbsaOSS)
-    - we also contribute to others, like Apache Spark
-- Research of new technologies and investigation of their usability in our Big Data platforms
-- Filling gaps in the big data eco-system for it to be fully usable by financial and other highly regulated industries
-- Presenting R&D work at commercial and academic conferences
-- Mentoring junior big data engineers
+-	Design and implementation of generic tools and platforms for some of the following:
+  - enabling simple data ingestion and consumption of corporate data lake
+  - various streaming use cases and products
+  - machine learning applications and pipelines
+-	Contribute to our projects, which can be found on [GitHub](https://github.com/AbsaOSS)
+  - we also contribute to others, like Apache Spark
+-	Research of new technologies and investigation of their usability in our Big Data platforms
+-	Filling gaps in the big data eco-system for it to be fully usable by financial and other highly regulated industries
+-	Presenting R&D work at commercial and academic conferences
+-	Mentoring junior big data engineers
```

Comment:
Are these intentional changes? The second-level bullet points are not indented anymore

Reply:
All fixed. Thank you. I did something with the whitespaces, I am not sure what.